bin/automated

```diff
@@ -6,8 +6,6 @@
 set -e
 
 # Variables
-SOURCE_BRANCH="master"
-ORIGIN_URL=`git config --get remote.origin.url`
 ORIGIN_CREDENTIALS=${ORIGIN_URL/\/\/github.com/\/\/$GH_TOKEN@github.com}
 COMMIT_MESSAGE=$(git log -1 --pretty=%B)
 TALKS="infra-2017 machine-learning-2017 front-2017"
```

Comment:
why don't you move talks to a directory (e.g. `talks`) so that you can replace this with something like
```
TALKS=$(ls landing/talks)
```
and you don't have to remember to update this every time a new talk is added?